gef-ui/templates/partials/review-decision.njk

```diff
@@ -0,0 +1,119 @@
+{% extends "index.njk" %}
+{% from "moj/components/banner/macro.njk" import mojBanner %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% block pageTitle%}
+    Review UKEF decision
+{% endblock %}
+
+{% block content %}
+
+    {% if errors %}
+        {{
+            govukErrorSummary({
+            titleText: "There is a problem",
+            errorList: errors.errorSummary,
+            attributes: {
+                'data-cy': 'error-summary'
+                }
+            })
+        }}
+    {% endif %}
+
+    {% set hide_back = 1 %}
+    {% include "includes/application-preview/summary.njk" %}
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full-from-desktop">
+            {{ govukBackLink({
+                text: "Back",
+                href: "/gef/application-details/" + applicationId,
+                attributes: {
+                'data-cy': 'back-link'
+                }
+            }) }}
+        </div>
+    </div>
+
+    {% if isUkefReviewAvailable %}
+    
+    <header>
+        {% set bannerHtml %}
+            <h2 data-cy="ukef-review-decision-heading">Review UKEF decision</h2>
+            {% if applicationStatus == 'UKEF_APPROVED_WITHOUT_CONDITIONS' %}
+                <p>UKEF has approved your application without conditions.</p>
+            {% elif applicationStatus == 'UKEF_APPROVED_WITH_CONDITIONS' %}
+                <p>UKEF has approved your application with the following conditions:</p>
+                <p>{{ ukefDecision[0].text }}</p>
+            {% elif applicationStatus == 'UKEF_REFUSED' %}
+                <p>The manual inclusion application was rejected for the following reasons:</p>
```

Comment:
you need to use the ` | safe`  pipe format for these sections:
 `{{ ukefDecision[0].text | safe}}`
Because users can type anything in the decision box, including tags or they can add new rows